scss/components/_commercial.scss

```diff
@@ -41,17 +40,17 @@
 }
 
 .list-letters-three {
-  margin: 32px 0 0 114px;
+  margin: 32px 0 0 46%;
```

Comment:
46% - bad value. Use % only for 100%, 50% etc

Reply:
Fixed